source/javascripts/components/GuidedOnboarding/Collapsible/Collapsible.tsx

```diff
@@ -17,6 +18,20 @@ const Collapsible = ({ open = false, children, title }: CollapsibleProps): JSX.E
   );
   const ref = useRef<HTMLDivElement>(null);
 
+  const trackOpenClose = useTrackingFunction((isOpen: boolean) => ({
+		event: isOpen ? "Guided Onboarding Displayed" : "Guided Onboarding Closed",
+		payload: {
+			step: "Configure your workflows"
```

Comment:
The [actual step name is all lowercase](https://github.com/bitrise-io/bitrise-workflow-editor/pull/699/files#diff-b8b3c93f504439df0a4329cfd4cfab802e47045e0b676c0c7a9f1b1256cb3201R21). I don't think the case actually matters, but my sense is that we shouldn't hardcode this, but rather determine it from current state and/or context. I don't totally have my head wrapped around how the various components fit together, so I'm not sure how feasible that is.

If it's not feasible, then I'm not sure what value there is in including the step in the tracking event payload at all, since it seems like it will only be correct (some of the time?) by coincidence.

Reply:
So inside the workflow editor there is only one active step. The other steps are in the component used in the main website. In @cdhenley219-bit's PR is doing what you describe as the step will actually change. 
https://github.com/bitrise-io/bitrise-website/pull/8245/files#diff-e092b95df78e1898b8fbf55ca66ec0f9161e73203d186aeb92e7d94c856dc589R79

Reply:
@jakerobb I refactored this a bit after thinking about it. Now the steps are controlled from the outermost scope and tracking the step name will be based on the current active step. For now the active step never changes in the workflow editor but these changes will make it easier to do that in the future.